bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/service/api/EventListener.java

```diff
@@ -12,29 +12,56 @@
  */
 package org.openhab.binding.samsungtv.internal.service.api;
 
+import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.smarthome.core.thing.ThingStatusDetail;
 import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.io.net.http.WebSocketFactory;
 
 /**
  * Interface for receiving data from Samsung TV services.
  *
  * @author Pauli Anttila - Initial contribution
+ * @author Arjan Mels - Added methods to put/get configuration
  */
+@NonNullByDefault
 public interface EventListener {
     /**
      * Invoked when value is received from the TV.
-     * 
+     *
      * @param variable Name of the variable.
-     * @param value Value of the variable value.
+     * @param value    Value of the variable value.
```

Comment:
see below

Reply:
Sure, I reformatted all files with the ESH codestyle formatter properly enabled.